testsuite/features/init_clients/sle_client.feature

```diff
@@ -3,12 +3,13 @@
 
 Feature: Register a traditional client
   In order to register a traditional client to the SUSE Manager server
-  As the root user
-  I want to call rhnreg_ks
+  I want to create, parametrize and run boostrap script from proxy
 
   Scenario: Register a traditional client
-    When I register using "1-SUSE-DEV-x86_64" key
-    Then I should see "sle_client" in spacewalk
+    Given I am authorized
+    When I bootstrap traditional client "sle_client" using bootstrap script with activation key "1-SUSE-DEV-x86_64" from the proxy
+    And I follow the left menu "Systems > Overview"
+    And I wait until I see the name of "sle_client", refreshing the page
```

Comment:
Given that we are bootstrapping using the script, so not using the WebUI.
It would be awesome if we can stay like that during the whole scenario.
What about check it using `Then I should see "sle_client" in spacewalk` as it was before?
We will also rid of the step of log in, which it use to be slow.

Reply:
Isn't that exactly what `I should see "sle_client" in spacewalk` was doing?

I think that "in spacewalk" is a poor way of saying "in the web UI".

I did not check, I'm just guessing here.

Reply:
@Bischoff @srbarrios Good point, I moved away from `I should see "sle_client" in spacewalk` during some debuging phase, but it shouldn't get to the final code. 

I looked into code definition for this step and we are still in web UI:
```
Then(/^I should see "([^"]*)" in spacewalk$/) do |host|
  steps %(
    Given I am on the Systems page
    Then I should see "#{host}" as link
    )
end
```

But I like the idea of staying in the CLI only. I will prepare solution with `spacecmd` command. According to my experience it could buy us a lot of time.

Reply:
Prepared, I am testing it now.

Reply:
Cool. But I would suggest to go further and take profit of those changes to change `in spacewalk` with something else, or just remove it.

Rationale:
* of course it's spacewalk, we're not speaking about Microsoft Office 😄 
* is it spacewalk, after all? Current name is Uyuni or SUSE Manager
* it's ultimately irrelevant, what is meant is "in the UI"

I suggest to completely remove these two words, or replace them with something that makes sense, not forgetting the documentation.

Reply:
@Bischoff It makes sense :smile:  OK, so what about following suggestion?

```I should see "([^"]*)" via spacecmd$```

We already used similar naming in the past, for example:

```And I wait until package "virgo-dummy" is installed on "ubuntu_ssh_minion" via spacecmd```

Reply:
Absolutely.

Reply:
Perfect, let's prepare it :-)